### ISSUE TITLE
Translate Outreachy sample tweets

### ISSUE THREAD

**Issue by sagesharp** [outreachy/creative-works-and-scripts#14]
This is a meta issue. We could use people translating [the sample tweets](https://github.com/sagesharp/outreachy/blob/master/creative-works/sample-tweets.txt) and [sample email](https://github.com/sagesharp/outreachy/blob/master/creative-works/sample-email.txt) about the Outreachy internships to other languages. 

We know we get many applicants from:
- [ ] Brazil - Brazilian Portuguese is needed
- [ ] India
- [ ] Russia
- [ ] Germany

You can see statistics on where applicants came from on [this blog post](https://www.outreachy.org/blog/outreachy-statistics-december-march-2017-applicants/)

**Comment by yogmel** [comment]
Hi @sagesharp, I can translate the texts to Portuguese and send them here to review.

**Comment by sagesharp** [comment]
@yogmel Yes, please! You can add a file called sample-tweets-LANGUAGE.txt in the same directory. Feel free to send a PR and reference this issue.

**Comment by dnabanita7** [comment]
Hello @sagesharp,I am from India.So,which language shall I translate to?

**Comment by sagesharp** [comment]
@Naba7 Which languages do you speak? I think Hindi is one of the most common languages. Others are listed here: https://en.m.wikipedia.org/wiki/Languages_of_India

**Comment by dnabanita7** [comment]
hey! sorry for late reply. Yes,I can speak Hindi and Odia

**Comment by dnabanita7** [comment]
So shall I translate it to Hindi or Odia?

**Comment by gconnect** [comment]
I would like to contribute to any of the above listed language translations but I don't understand/speak any of those languages. Is it okay if I rely on google translation to translate it?

**Comment by sagesharp** [comment]
@gconnect Please only translate to a language you speak. Google translate is often not accurate, especially when it comes to cultural nuances like how to translate the word "genderqueer". You're welcome to translate resources to a different language than the ones listed above.

**Comment by krishna7860** [comment]
this issue is still active can i contribute to convert it in hindi @sagesharp

**Comment by winnywine** [comment]
i would like to translate it in french if accepted 
just requesting @sagesharp  i will be waiting  if yes i will  
thank you

**Comment by SammyJoel** [comment]
Hi @sagesharp I am from Nigeria, and I would like to translate the sample tweets and sample email about the Outreachy internships 2024 to Hausa, a language spoken among 80 million people, both among Nigerians and none Nigerians.
